src/rf/apps/workers/management/commands/start-worker-process.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not a big deal, but seems like the convention for management command names is to omit spaces.